File: Praktikum 7/Python/PRAK601-2310817210004-Allano Lintang Ertantora.py
rows, cols = map(int, input().split())
matrix = list(map(int, input().split()))
matrix = [matrix[i:i+cols] for i in range(0, len(matrix), cols)]
for i in range(rows):
    for j in range(cols):
        print(matrix[i][j], end=" ")
    print()

File: Praktikum 7/C/PRAK601-2310817210004-Allano-Lintang-Ertantora.c
#include <stdio.h>

int main() {
    int rows, cols, i, j;
    scanf("%d %d", &rows, &cols);
    int matrix[rows][cols];
    for(i = 0; i < rows; i++) {
        for(j = 0; j < cols; j++ ){
            scanf("%d", &matrix[i][j]);
        }
    }for(i = 0; i < rows; i++){
        for(j = 0; j < cols; j++){
            printf("%d ", matrix[i][j]);
        }printf("\n");
    }
    return 0;
}